drivers/modem/modem_cellular.c

```diff
@@ -2350,6 +2442,9 @@ MODEM_CHAT_SCRIPT_DEFINE(sqn_gm02s_periodic_chat_script,
 		.ppp = &MODEM_CELLULAR_INST_NAME(ppp, inst),                                       \
 	};                                                                                         \
                                                                                                    \
+	MODEM_PIPELINK_DT_DEFINE(DT_DRV_INST(inst), shell);                                        \
+	MODEM_PIPELINK_DT_DEFINE(DT_DRV_INST(inst), gnss);                                         \
```

Comment:
It would be nice to have at least a reference of how things are meant to be defined in the devicetree.